project/settings/dev.py

```diff
@@ -49,3 +52,8 @@ def __mod__(self, other):
 
 
 TEMPLATES[0]['OPTIONS']['string_if_invalid'] = InvalidStringError("%s")
+
+if AWS_ACCESS_KEY_ID:
+    THUMBNAIL_DEBUG = True
```

Comment:
Je ten DEBUG bezpečný? Chceme ji na ostrém serveru?

Reply:
To je ale na dev settings.

Reply:
aha, jsem idiot :)